support/oidc-discovery-provider/handler_test.go

```diff
@@ -234,22 +234,44 @@ func TestHandlerHTTPInsecure(t *testing.T) {
 
 func TestHandlerHTTP(t *testing.T) {
 	testCases := []struct {
-		name    string
-		method  string
-		path    string
-		jwks    *jose.JSONWebKeySet
-		modTime time.Time
-		code    int
-		body    string
+		name         string
+		overrideHost string
+		method       string
+		path         string
+		jwks         *jose.JSONWebKeySet
+		modTime      time.Time
+		code         int
+		body         string
 	}{
 		{
 			name:   "GET well-known",
 			method: "GET",
 			path:   "/.well-known/openid-configuration",
 			code:   http.StatusOK,
 			body: `{
-  "issuer": "https://localhost",
-  "jwks_uri": "https://localhost/keys",
+  "issuer": "https://domain.test",
```

Comment:
since this test is using real `DomainAllowlist`, may we have a test case that provides a unicode or punicode?

Reply:
Good idea.